spec/public-api.yaml

```diff
@@ -265,13 +265,11 @@ components:
           default: 0.9
           description: If the detector's prediction is below this confidence threshold,
             send the image query for human review.
-        config_name:
+        pipeline_config:
```

Comment:
the main change of this PR. Question: should we just keep the field as "config_name". Also, are there any customers we know of who use the `config_name` field?

Reply:
None of our new customers have started using it as far as I know. Lkq would be the only one who could be using it